packages/admin/cms-admin/src/dam/FileForm/CropSettingsFields.tsx

```diff
@@ -35,33 +36,58 @@ export function CropSettingsFields({ disabled }: Props): JSX.Element {
     return (
         <Container>
             <FormSection title={<FormattedMessage id="comet.dam.file.cropSettings.sectionTitle" defaultMessage="Crop/Focus settings" />}>
-                <FieldContainer>
-                    <FormControlLabel
-                        control={<Switch checked={focalPoint === "SMART"} onChange={handleSmartFocalPointChange} />}
-                        label={<FormattedMessage id="comet.dam.file.smartFocusPoint" defaultMessage="Smart focus point" />}
-                    />
-                    <Box mt={2} pl={2}>
-                        <Typography variant="body2" paragraph>
-                            <FormattedMessage
-                                id="comet.dam.file.croppingInfoText"
-                                defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
-                            />
-                        </Typography>
-                        <Typography variant="body2">
-                            <FormattedMessage
-                                id="comet.dam.file.focusPointInfoText"
-                                defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
-                            />
-                        </Typography>
-                    </Box>
+                <FieldContainer
+                    fullWidth
+                    helperText={
+                        <Box>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.croppingInfoText"
+                                    defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
+                                />
+                            </Typography>
+                            <Typography variant="caption" color="text.secondary" paragraph>
+                                <FormattedMessage
+                                    id="comet.dam.file.focusPointInfoText"
+                                    defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
+                                />
+                            </Typography>
+                        </Box>
```

Comment:
Generally `helperText` should already be styled correctly and should therefor not need the `Typography`. 
You may need to add the spacing between the two lines though but this could simply be `<br /><br />`. 

```suggestion
                        <>
                            <FormattedMessage
                                id="comet.dam.file.croppingInfoText"
                                defaultMessage="Cropping selects the maximum visible area. Depending on the aspect ratio, the image may be cropped further on the page."
                            />
                            <br />
                            <br />
                            <FormattedMessage
                                id="comet.dam.file.focusPointInfoText"
                                defaultMessage="The focus point marks the most important part of the image, which is always visible. Choose it wisely."
                            />
                        </>
```

Reply:
6e87721e8e8f0bb55111891b805e3912a6e911a4